packages/sdk/src/services/RemoteConnection/ConnectionManager/startConnection.ts

```diff
@@ -92,11 +105,33 @@ export async function startConnection(
 
   // if we are on desktop browser
   const qrCodeOrigin = state.platformManager?.isSecure() ? '' : '&t=q';
+  const sdkVersion = packageJson.version;
+  const { iconUrl, name, url } = options.dappMetadata || {};
+  const platformType = state.platformManager?.getPlatformType();
+
+  let base64OriginatorInfo: string | undefined;
+
+  if (
+    !(typeof window === 'undefined' || typeof window.location === 'undefined')
+  ) {
```

Comment:
```suggestion
if (typeof window?.location !== 'undefined') {
```

Reply:
This had ot be removed otherwise react native will fail you need to specifically test for window as I did.